Packages/com.unity.inputsystem/InputSystem/Editor/Internal/AdvancedDropdown/AdvancedDropdownDataSource.cs

```diff
@@ -118,7 +118,7 @@ private void BuildSearchableElements(AdvancedDropdownItem item)
         {
             if (!item.children.Any())
             {
-                if (!item.IsSeparator())
+                if (!item.IsSeparator() && item.searchableName.Length > 0)
```

Comment:
That's it basically. We would like not to search in non-searchable items because they end up as empty lines in the displayed list in the end.